packages/core/src/types.ts

```diff
@@ -272,7 +272,15 @@ export type Composition = {
   componentSettings?: ExperienceComponentSettings;
 };
 
-export type DesignTokensDefinition = { [key: string]: string | DesignTokensDefinition };
+export type DesignTokensDefinition = {
+  spacing?: Record<string, string>;
+  sizing?: Record<string, string>;
+  colors?: Record<string, string>;
+  borders?: Record<string, { size: string; color: string }>;
```

Comment:
can you add one more property called `style: string`, also can you change `size` to `width`